tensorcircuit/templates/blocks.py

```diff
@@ -149,3 +149,25 @@ def example_block(
         for i in range(n):
             c.rx(i, theta=param[2 * j + 1, i])
     return c
+
+
+def qft(c: Circuit, *index: int) -> None:
+    """
+    This function applies quantum fourier transformation (QFT) to the selected circuit lines
+
+    :param c: Circuit in
+    :type c: Circuit
+    :param *index: the indices of the circuit lines to apply QFT
+    :type *index: Tuple[int, int]
```

Comment:
the type of index can be documented as int or List[int], but not Tuple[int, int], as the latter only refer two cases with two ints

Reply:
also better return the circuit c instead of None for consistency with other template function

Reply:
ok